registries/operations-collector-registry.yaml

```
@@ -48,6 +48,9 @@ exporters:
     path: "../exporter/googleservicecontrolexporter"
 connectors:
 extensions:
+  healthagent:
+    gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/healthagent v0.0.0
```

This needs to match https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/blob/ee73a6b2f7c61121ff8799da61a107edcce1e376/extension/healthagent/go.mod#L1C10-L1C95

```suggestion
    gomod: github.com/GoogleCloudPlatform/opentelemetry-operations-collector/extension/healthagent v0.0.0
```